src/Drupal/Composer/DrupalInstallerPlugin.php

```diff
@@ -98,9 +98,9 @@ public function activate(Composer $composer, IOInterface $io) {
         );
 
         // Read user environment overrides.
-        $remote = getenv("COMPOSER_GIT_REMOTE");
+        $remote = isset(getenv("COMPOSER_GIT_REMOTE")) ? getenv("COMPOSER_GIT_REMOTE") : 'origin';
```

Comment:
Ah missed that so its just the environment variable setting  not being used so this could be a simpler change just to use that.

> On Nov 2, 2015, at 3:18 AM, Doug Green <notifications@github.com> wrote:
> 
> In src/Drupal/Composer/DrupalInstallerPlugin.php:
> 
> > @@ -98,9 +98,9 @@ public function activate(Composer $composer, IOInterface $io) {
> >          );
> >  
> >          // Read user environment overrides.
> > -        $remote = getenv("COMPOSER_GIT_REMOTE");
> > +        $remote = isset(getenv("COMPOSER_GIT_REMOTE")) ? getenv("COMPOSER_GIT_REMOTE") : 'origin';
> $this->get['remote'] is initialized to origin on line 96
> 
> —
> Reply to this email directly or view it on GitHub.
>